.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "Vendor/libmobilecoin-ios-artifacts"]
 	path = Vendor/libmobilecoin-ios-artifacts
-	url = https://github.com/mobilecoinofficial/libmobilecoin-ios-artifacts.git
```

Comment:
this should be reverted

Reply:
will update when libmobilecoin-ios-artifacts published

Reply:
ah, looks like it's not working... will revert